GiEnJul/ClientApp/src/components/admin/suggestedConnections/RecipientTable.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useEffect, useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Box from "@material-ui/core/Box";
+import Collapse from "@material-ui/core/Collapse";
+import IconButton from "@material-ui/core/IconButton";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Typography from "@material-ui/core/Typography";
+import Paper from "@material-ui/core/Paper";
+import KeyboardArrowDownIcon from "@material-ui/icons/KeyboardArrowDown";
+import KeyboardArrowUpIcon from "@material-ui/icons/KeyboardArrowUp";
+import { RecipientType } from "./Types";
+import ApiService from "../../../common/functions/apiServiceClass";
+import { getGender } from "../../../common/functions/GetGender";
+import { capitalize } from "@material-ui/core";
+
+const api = new ApiService();
+
+type RecipientRowProps = {
+  recipient: RecipientType;
+  onClick: any;
+  selected: boolean;
+};
+
+function Row(props: RecipientRowProps) {
+  const { recipient } = props;
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <>
+      <TableRow
+        onClick={() => {
+          props.onClick(recipient.rowKey, recipient.partitionKey);
+          setOpen(!open);
+        }}
+        /*
+        hover
+        selected={props.selected}
+        aria-checked={props.selected}
+        
+        style={props.selected
+            ? { color: "blue", backgroundColor: "ActiveBorder" }
+            : {}
+        }
+        */
```

Comment:
Is this comment supposed to be here? If yes: it would be nice to have a comment why this is still here.

Reply:
Removed